app/(feed)/page.tsx

```diff
@@ -1,16 +1,25 @@
 import FeedPage from "@/components/feed";
+import FilterFeedButtons from "@/components/feed/FilterFeedButtons";
 import PageTitle from "@/components/ui/PageTitle";
 import { getFeedItems } from "@/lib/feed";
 import { getCurrentUser } from "@/lib/getUser";
+import { getSavedFeedItemsList } from "@/lib/savedFeedItems";
 
 const Feed = async () => {
   const data = await getFeedItems();
   const currentUser = await getCurrentUser();
+  const savedItemIds = await getSavedFeedItemsList(currentUser.id);
 
   return (
     <div>
       <PageTitle title="Feed" />
-      <FeedPage data={data} currentUser={currentUser} showLinks />
+      <FilterFeedButtons />
+      <FeedPage
+        data={data}
+        currentUser={currentUser}
+        savedItemIds={savedItemIds}
```

Comment:
Why do we have to get the save item IDs on the feed if all the saved stuff is on a separate page? can you remove this / make `savedItemIds` optional

Reply:
I have the saved item IDs coming in to set the state of the Save button on each feed item. Do I need to go about this differently, or is this acceptable for what I'm using it for?

Reply:
I ended up making savedItemIds optional.